pom.xml

```diff
@@ -72,17 +72,26 @@
         <artifactId>google-cloud-aiplatform</artifactId>
         <version>0.2.1-SNAPSHOT</version><!-- {x-version-update:google-cloud-aiplatform:current} -->
       </dependency>
+      <dependency>
+        <groupId>com.google.api.grpc</groupId>
+        <artifactId>proto-google-cloud-aiplatform-v1</artifactId>
+        <version>0.2.1-SNAPSHOT</version><!-- {x-version-update:proto-google-cloud-aiplatform-v1:current} -->
+      </dependency>
       <dependency>
         <groupId>com.google.api.grpc</groupId>
         <artifactId>proto-google-cloud-aiplatform-v1beta1</artifactId>
         <version>0.2.1-SNAPSHOT</version><!-- {x-version-update:proto-google-cloud-aiplatform-v1beta1:current} -->
       </dependency>
+      <dependency>
+        <groupId>com.google.api.grpc</groupId>
+        <artifactId>grpc-google-cloud-aiplatform-v1</artifactId>
+        <version>0.2.1-SNAPSHOT</version><!-- {x-version-update:grpc-google-cloud-aiplatform-v1:current} -->
+      </dependency>
       <dependency>
         <groupId>com.google.api.grpc</groupId>
         <artifactId>grpc-google-cloud-aiplatform-v1beta1</artifactId>
-        <version>0.2.1-SNAPSHOT</version><!-- {x-version-update:grpc-google-cloud-aiplatform-v1beta1:current} -->
+        <version>0.2.1-SNAPSHOT</version><!-- {x-version-update:grpc-google-cloud-aiplatform-v1:current} -->
```

Comment:
Please restore this one. When the library is promoted to 1.0.0, the v1beta1 artifacts will stay at 0.x

```suggestion
        <version>0.2.1-SNAPSHOT</version><!-- {x-version-update:grpc-google-cloud-aiplatform-v1beta1:current} -->
```